### DISCUSSION TITLE
可以默认监听全部应用程序吗

### DISCUSSION THREAD

**Discussion by include-nan** [shion-app/shion#50] [Ideas]
不想手动挨个去添加，而且还必须是应用在运行时才能添加，希望能默认监听全部运行的程序

**Comment by hanaTsuk1** [comment]
嫌麻烦就把所有盘都添加到白名单

https://shion.app/zh/guide/general#%E8%87%AA%E5%8A%A8%E7%9B%91%E5%90%AC%E8%83%BD%E4%B8%8D%E8%83%BD%E8%87%AA%E5%8A%A8%E6%B7%BB%E5%8A%A0%E7%A8%8B%E5%BA%8F
